### ISSUE TITLE
openexrcore/encoding.c: warning: format specifies type 'unsigned long' but the argument has type 'size_t' 

### ISSUE THREAD

**Issue by claudeha** [AcademySoftwareFoundation/openexr#1529]
When cross-compiling openexr-3.2.0-rc4 from Debian Bookworm x86_64 to Windows aarch64 and armv7 using llvm-mingw, I get a single warning for each:

aarch64:
```
/home/claude/opt/windows/posix/aarch64/src/openexr-3.2.0-rc4/src/lib/OpenEXRCore/encoding.c:35:13: warning: format specifies type 'unsigned long' but the argument has type 'size_t' (aka 'unsigned long long') [-Wformat]
            exr_compress_max_buffer_size (encode->packed_bytes));
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

armv7:
```
/home/claude/opt/windows/posix/armv7/src/openexr-3.2.0-rc4/src/lib/OpenEXRCore/encoding.c:35:13: warning: format specifies type 'unsigned long' but the argument has type 'size_t' (aka 'unsigned int') [-Wformat]
            exr_compress_max_buffer_size (encode->packed_bytes));
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

(I don't have any ARM Windows machines to test with so I don't know how significant this is...)

**Comment by cary-ilm** [comment]
@claudeha, this should be addressed by #1534, can you confirm?

**Comment by cary-ilm** [comment]
Closing for now since #1534 should resolve it, but feel free to reopen or submit a new issue if something is still amiss.
